ui/desktop/src/components/Modal.tsx

```diff
@@ -46,23 +46,29 @@ export default function Modal({
       }
     };
 
-    // Add event listener
+    // Add event listener for Escape key
     document.addEventListener('keydown', handleEscKey);
 
+    // Add overflow-hidden to body to prevent scrolling background
+    document.body.style.overflow = 'hidden';
```

Comment:
Awesome. This works well as a simple fix to lock scrolling because we only use nested scroll elements. If we ever do a full bleed scrolling window we'd need to do some management of `scrollTop`